.pre-commit-config.yaml

```diff
@@ -1,33 +1,54 @@
 exclude: '(^docs/conf.py|^user_data/External_tables/|^docs/references/|^pyuvdata/data/|^pyuvdata/uvdata/src/|^pyuvdata/tests/Line_Profiler_full.out|.codecov.yml)'
```

Comment:
just randomly noticed that you have `user_data/External_tables/` in here. I imagine this entered when I copied over the pre-commit file from 21cmFAST 😬

Reply:
you don't _want_ copy pasta'd code floating around?

Reply:
Good catch, thanks! Fixing it now.